Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
Changed the name from `boundingVolume` to `contentsBoundingVolume` in https://github.com/AnalyticalGraphicsInc/cesium/commit/f8a628e742cf0ef60559d505c6de154c6853aa75.